### PR TITLE
Update wine-wechat.desktop for better integration

### PR DESCRIPTION
**Pull request by bryango**
- `StartupWMClass=wechat` for better theming
- Add `Actions` for right click menu

No need for separate `wine-wechat-cfg.desktop` & `wine-wechat-dir.desktop`, but they are kept so that old users can find them more easily.

### CODE REVIEW

File: archlinuxcn/wine-wechat/wine-wechat.desktop [hunk 1]
```diff
@@ -11,3 +11,19 @@ Terminal=0
 Type=Application
 Categories=Network;InstantMessaging;
 Keywords=wx;wechat;weixin
+StartupWMClass=wechat
+Actions=Configure;Directory;
+
+[Desktop Action Configure]
+Exec=env WINEDEBUG=-all wechat --config
+Name=Configure Wine
```
Comment (lilydjwg) [inline archlinuxcn/wine-wechat/wine-wechat.desktop:19]: This message is inaccurate. It's for configuring "Wine Wechat".

File: archlinuxcn/wine-wechat/wine-wechat.desktop [hunk 2]
```diff
@@ -11,3 +11,19 @@ Terminal=0
...
+Name[zh_CN]=Wine 微信配置
+Comment=Run winecfg for Wine WeChat
+Comment[zh_CN]=为 Wine 微信运行 winecfg
+
+[Desktop Action Directory]
+Exec=env WINEDEBUG=-all wechat --dir
+Name=User Directory
```
Comment (lilydjwg) [inline archlinuxcn/wine-wechat/wine-wechat.desktop:26]: ditto.

File: archlinuxcn/wine-wechat/wine-wechat.desktop [hunk 3]
```diff
@@ -11,3 +11,19 @@ Terminal=0
 Type=Application
 Categories=Network;InstantMessaging;
 Keywords=wx;wechat;weixin
+StartupWMClass=wechat
```
Comment (lilydjwg) [inline archlinuxcn/wine-wechat/wine-wechat.desktop:14]: It should be  "wechat.exe". How will this help with theming?

**Review by lilydjwg — CHANGES_REQUESTED** [review]

File: archlinuxcn/wine-wechat/wine-wechat.desktop (hunk 1 above)
Comment (bryango) [reply archlinuxcn/wine-wechat/wine-wechat.desktop:19]: Sure, but technically it should be "configure wine (winecfg) for wechat", not "configure wechat", which sounds like configuring wechat itself (e.g. opening its settings). But I agree with you. Please suggest how to phrase this more concisely.

File: archlinuxcn/wine-wechat/wine-wechat.desktop (hunk 2 above)
Comment (bryango) [reply archlinuxcn/wine-wechat/wine-wechat.desktop:26]: ditto; please suggest how to phrase this accurately & concisely.

I've allowed edits by maintainers, so please feel free to change it as you like; I might not be able to respond in time...

**Review by bryango — COMMENTED** [review]
I've allowed edits by maintainers, so please feel free to change it as you like; I might not be able to respond in time...

BTW, I couldn't find the source for `wine-wechat.tar.zst`... Could you point me to it? Thank you very much!

**Comment by lilydjwg** [comment]
This is no source code for `wine-wechat.tar.zst`. It's a wine prefix with wechat installed. You can examine it from the package.

**Pull request closed without merging**